install_config/configuring_sdn.adoc

```diff
@@ -164,6 +164,181 @@ Check VNIDs by running:
 $ oc get netnamespace
 ----
 
+[[renumbering-sdn-networks]]
+== Renumbering the SDN Network
+
+It is difficult to change the networking configuration of an
+{product-title} cluster after it has been deployed, but if absolutely
+necessary, it can be done.
+
+=== Expanding the Cluster IP Range By Adding Nodes
+
+The easiest reconfiguration involves changing the
+`*clusterNetworkCIDR*` to make room to add additional nodes, while
+leaving existing Nodes and Pods untouched.
```

Comment:
Can we make this a little more clear, perhaps by saying something like:

"In order to use this method, the new range must completely contain the original clusterNetworkCIDR."